### DISCUSSION TITLE
Can this be used for impartus of other institutes?

### DISCUSSION THREAD

**Discussion by Yashpandey4** [iamkroot/ilc-scraper#51] [General]
For example, IITD courses have URLs of the type: https://a.impartus.com/ilc/#/course/675659/456
I wonder what modifications need to be made to make this app interoperable

**Comment by DevDSVik** [comment]
It is only downloading the lectures in the lectures tab. How to download flipped lectures?

**Reply by shouvik1984** [reply]
Flipped lectures are on Youtube... you can download via any Youtube Downloader
